cmd/gateway/azure/gateway-azure.go

```diff
@@ -130,6 +130,10 @@ func (g *Azure) NewGatewayLayer(creds auth.Credentials) (minio.GatewayLayer, err
 		}
 	}
 
+	if endpoint == fmt.Sprintf("%s.blob.%s", creds.AccessKey, storage.DefaultBaseURL) {
+		// If, by mistake, user provides endpoint as accountname.blob.core.windows.net
```

Comment:
doesn't `<ACCOUNT_NAME>.blob.core.windows.net` work?

Reply:
Is `<ACCOUNT_NAME>.blob.core.windows.net` legal?

Reply:
Yes it is legal @balamurugana